nexus/tests/integration_tests/metrics.rs

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use dropshot::test_util::ClientTestContext;
+use dropshot::ResultsPage;
+use nexus_test_utils::resource_helpers::objects_list_page_authz;
+use omicron_common::backoff;
+use oximeter::types::Datum;
+use oximeter::types::Measurement;
+
+pub async fn query_for_metrics_until_they_exist(
+    client: &ClientTestContext,
+    path: &str,
+) -> ResultsPage<Measurement> {
+    backoff::retry_notify(
```

Comment:
nit: not really sure why we'd backoff here -- we're not overloading anything

Reply:
Switched to just a sleep, cleaned up a bit of this retry logic in: 6837d177af53c26aadc4647305bfa57a66b20269